src/sentry/models/dashboard.py

```diff
@@ -3,6 +3,19 @@
 
 from sentry.db.models import FlexibleForeignKey, Model, sane_repr
 from sentry.db.models.fields.bounded import BoundedBigIntegerField
+from sentry.db.models.fields.jsonfield import JSONField
+
+
+class DashboardProject(Model):
```

Comment:
Is it necessary to explicitly define a through table? I know Django can auto-create one for you, but I noticed our M2M fields always define a table explicitly.

Is it to be more explicit in controlling values such as table name and constraints?

Reply:
> Is it to be more explicit in controlling values such as table name and constraints? 

Partly this, and partly a convention from old django.

Reply:
If we're not placing any logic on the junction table and we can get the table name conventions on the generated model, I don't see a reason to have a formally defined junction model.

Reply:
One useful thing with declaring the table explicitly is that you can add a `date_added` column to it, which is sometimes useful for investigations.